aquarius/events/events_monitor.py

```diff
@@ -163,12 +166,13 @@ def run_monitor(self):
                     return
 
                 self.process_current_blocks()
-                if first_update:
-                    self._updater.do_update()
-                    first_update = False
-
-                self._updater.process_pool_events()
-                self._update_purgatory_list()
+                if bool(int(os.getenv('PROCESS_POOL_EVENTS', 1)) == 1):
+                    if first_update:
+                        self._updater.do_update()
+                        first_update = False
+                    self._updater.process_pool_events()
+                if bool(int(os.getenv('PROCESS_PURGATORY', 1)) == 1):
```

Comment:
use `UPDATE_ALL_PURGATORY`, or keep this and replace `UPDATE_ALL_PURGATORY` env var